QEfficient/cloud/infer.py

```diff
@@ -69,7 +70,7 @@ def main(
         f"qpc_{num_cores}cores_{batch_size}BS_{prompt_len}PL_{ctx_len}CL_"
         + f"{len(device_group)}"
         + "devices"
-        + ("_mxfp6" if mxfp6 else "_fp16")
+        + ("_mxfp6_mxint8" if mxfp6 and mxint8 else "_mxfp6" if mxfp6 else "_fp16")
```

Comment:
is fp16 + mxint8 KV not a valid option?

Reply:
It is, CustomIO $KVs precision can still be in mxint8. right?

Reply:
FP16 is default in all 3 cases here.